pages/content/reports/2024/index.md

```diff
@@ -0,0 +1,232 @@
+
+In 2024, Open Web Docs:
+
+- Was again the organization that contributed the greatest number of pull requests (PRs) merged to the [mdn/content](https://github.com/mdn/content) (14.1%) and [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) (52.4%) repositories.
+- Provided 27% of all pull request reviews to the mdn/content repository and 52% of reviews to mdn/browser-compat-data.
+- Started to become a maintainer of the [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features) repo and contributed 14.8% of merged pull requests.
+- Completed major technical writing and documentation engineering projects which were determined and guided by OWD’s [Steering Committee](https://github.com/openwebdocs/project/blob/main/steering-committee/membership-expectations.md). This included among other things:
+  - [Collecting browser compatibility data whenever browsers release a new beta version](#collecting-browser-compatibility-data-whenever-browsers-release-a-new-beta-version)
+  - [Documentation for interoperable and widely available features](#documentation-for-interoperable-and-widely-available-features)
+  - [W3C WebDX web-features mapping of BCD keys into baseline features](#w3c-webdx-web-features-mapping-of-bcd-keys-into-baseline-features)
+  - [Replacing all "true" and "null" versions with real version numbers in BCD](#replacing-all-true-and-null-versions-with-real-version-numbers-in-bcd)
+  - [Web Security docs for MDN](#web-security-docs-for-mdn)
+  - [Updated Fetch documentation](#updated-fetch-documentation)
+  - [WebView compatibility data](#webview-compatibility-data)
+- Secured a second round of [investment from the Sovereign Tech Fund](https://openwebdocs.org/content/posts/sovereign-tech-fund/) to implement improved tooling and maintenance of the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) project and development of the [openwebdocs/mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) tool.
+
+Previous Open Web Docs Impact and Transparency Reports: [2023](https://openwebdocs.org/content/reports/2023/), [2022](https://openwebdocs.org/content/reports/2022/), [2021](https://openwebdocs.org/content/reports/2021/).
+
+<div class="callout">
+<p>Open Web Docs is a non-profit, strongly community-focused open source collective that uses its donations to employ a group of technical writers who are dedicated to writing and improving documentation for the web platform.</p>
```

Comment:
Minor, but you might be able to drop the `<p>` HTML tagging here by just adding an empty line:

```suggestion

Open Web Docs is a non-profit, strongly community-focused open source collective that uses its donations to employ a group of technical writers who are dedicated to writing and improving documentation for the web platform.
```